Base/Logger/Logger.Static/FileLogger.hpp

```diff
@@ -28,13 +28,13 @@ class FileLogger : ILogger
     static FileLogger& getInstance();
 
     /**
-    * @brief Initialization of the current class object
-    * @param filename std::string name to write data
-    * @param output LogOutput - the place where the data will be displayed
-    */
+     * @brief Initialization of the current class object
+     * @param filename std::string name to write data
+     * @param output LogOutput - the place where the data will be displayed
+     */
```

Comment:
These spaces are added by the formatter?

Reply:
No, I did it for uniformity with others comments of the same type.

Reply:
Can we close this subject?

Reply:
I think, yes